### PR TITLE
Warning about authorization of output event-type is not that of query

### PR DESCRIPTION
**Pull request by thomasabraham**
Warning about authorization of output event-type is not that of query

### CODE REVIEW

File: client/Pages/EventTypeDetails/View.elm [hunk 1]
```diff
@@ -770,7 +770,16 @@ authTab appsInfoUrl usersInfoUrl teamsInfoUrl eventType =
         Just authorization ->
             div [ class "dc-card auth-tab" ]
                 [ div [ class "auth-tab__content" ]
-                    [ AccessEditor.viewReadOnly
+                    if isQueryOutput then
+                      [ warningMessage
+                            "This is the Authorization of the output event-type!"
+                            "Authorization of queries are not visible in Nakadi-UI"
```
Comment (a1exsh) [inline client/Pages/EventTypeDetails/View.elm:776]: I think this begs the next question: where is that visible?  Can we have a link to the docs or something like that?

File: client/Pages/EventTypeDetails/View.elm [hunk 2]
```diff
@@ -218,7 +218,15 @@ detailsLayout typeName eventType model =
                     ]
                 , tabs tabOptions (Just tab) <|
                     List.concat
-                        [ [ ( SchemaTab
+                        [ if isQueryOutput then
```
Comment (a1exsh) [inline client/Pages/EventTypeDetails/View.elm:221]: Why do we change the order of tabs?

**Comment by thomasabraham** [comment]
Sorry @a1exsh This is work in progress, I'm not able to build locally.

File: client/Pages/EventTypeDetails/View.elm (hunk 2 above)
Comment (thomasabraham) [reply client/Pages/EventTypeDetails/View.elm]: Reverted this

**Comment by tor-vs-floki** [comment]
👍

**Comment by thomasabraham** [comment]
:+1:

**Pull request merged**